apps/documents/serializers.py

```diff
@@ -68,6 +68,15 @@ class Meta:
             exclude = ('shipment',)
         meta_fields = ('presigned_s3',)
 
+    def validate_shipment_id(self, shipment_id):
```

Comment:
We don't need a shipment_id in the DocumentCreateSerializer if it is in the path. Remove the field and parse the context in the serializer create() method instead.

Reply:
Yes, I agree, James brought that up the other day and i thought it could be helpful to compare the two `shipment_id` since it represents the edge case in which an authenticated user tries to temper the the data on the client side. That's why i made a `LOG.warning` to track such kind of behavior.